server/src/main/scala/io/delta/standalone/internal/DeltaSharingHistoryManager.scala

```diff
@@ -21,7 +21,7 @@ import java.sql.Timestamp
 
 import io.delta.standalone.internal.actions.CommitMarker
 import io.delta.standalone.internal.util.FileNames
-import io.delta.standalone.storage.LogStore
+import io.delta.storage.LogStore
```

Comment:
This is changed because of the upgrade of dependency on delta-standalone to 0.5.0